frontend/src/views/Org/components/CreateOrgModal.tsx

```diff
@@ -0,0 +1,99 @@
+import { FC } from "react";
+import { Controller, useForm } from "react-hook-form";
+import { useRouter } from "next/router";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
+
+import { useNotificationContext } from "@app/components/context/Notifications/NotificationProvider";
+import { Button, FormControl, Input, Modal, ModalContent } from "@app/components/v2";
+import { useCreateOrg } from "@app/hooks/api";
+
+const schema = yup
```

Comment:
Can you please change to zod. We are moving towards `zod` from yup.  As backend also uses zod we wanna keep one validation lib through out infisical